recommender/engine_test.go

```diff
@@ -68,60 +61,46 @@ func (dm dummyNetworkMapper) MapNetworkPerf(vm productinfo.VmInfo) (string, erro
 // utility VmRegistry for mocking purposes
 type dummyProductInfo struct {
 	// test case id to drive the behaviour
-	TcId int
+	TcId string
 }
 
 func (d *dummyProductInfo) Start(ctx context.Context) {
 
 }
 func (d *dummyProductInfo) GetAttrValues(provider string, attribute string) ([]float64, error) {
 	switch d.TcId {
-	case 1:
-		// 3 values between 10 - 20
+	case "3 values between 10-20":
```

Comment:
Instead of using strings here i'd recommend defining constants / variables for these cases;  it's easier to maintain variable names than "free" strings